app/views/posts/index.html.erb

```diff
@@ -1,6 +1 @@
-<h1>Welcome to post index page</h1>
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= post.title %></li>
-  <% end %>
-</ul>
+<h1>Welcome to post index page</h1>
```

Comment:
- [ ] Kindly design the `Post#Index` page to look identical to that of the wireframe provided for this project. When I visit the `Post#Index` page, this is what is rendered 🖱️:

![image](https://github.com/ridabensalem/blog-app/assets/77106962/3e4705e8-2f56-468a-ad54-edd75dbcaeca)

#### Please design the Post#Index Page to look identical to this wireframe below 💻 :

![image](https://github.com/ridabensalem/blog-app/assets/77106962/5f9d7012-b35f-485e-bf47-c097c083ca2e)

#### Also, please don't forget to use the `recent_comments` method that you have in your `app/models/post.rb` file to render the **_5 most recent comments_** for each post in the `Posts#Index` page. 👍 😎